uiowa/uiowa
# [ ITS ] Change "Article" content type label to "News Article"

replaces https://github.com/uiowa/uiowa/pull/8261
closes https://github.com/uiowa/uiowa/issues/8244

## To test
- Pull branch
- Sync its
- Check that the content type is named "News article" and that the help text changes as well
